project/BuildHelper.scala

```diff
@@ -242,7 +243,13 @@ object BuildHelper {
   )
 
   def nativeSettings = Seq(
-    nativeConfig ~= { _.withMultithreading(false) },
+    nativeConfig ~= { cfg =>
+      val os = System.getProperty("os.name").toLowerCase
+      // For some unknown reason, we can't run the test suites in debug mode on MacOS
```

Comment:
Note to self: Need to followup with the Scala Native team and mention this in the release notes